CHANGELOG.md

```diff
@@ -1,5 +1,18 @@
 # Changelog
 
+## [7.1.0](https://github.com/theforeman/puppet-tftp/tree/7.1.0) (2022-02-03)
+
+[Full Changelog](https://github.com/theforeman/puppet-tftp/compare/7.0.0...7.1.0)
+
+**Implemented enhancements:**
+
+- puppetlabs/stdlib: Allow 8.x [\#129](https://github.com/theforeman/puppet-tftp/pull/129) ([bastelfreak](https://github.com/bastelfreak))
+- Add Debian 11 support [\#125](https://github.com/theforeman/puppet-tftp/pull/125) ([ekohl](https://github.com/ekohl))
+
+**Closed issues:**
+
+- Disable warning and report when Puppet 6.23+ tries to manage more than 1000 files [\#127](https://github.com/theforeman/puppet-tftp/issues/127)
```

Comment:
there was no change in `puppet-tftp` for this, so feels wrong to call it out in the changelog?

Reply:
I debated this. It may be the first time we mention a closed issue in the changelogs. I thought it may be relevant to some users but you're right: it only made changes in puppet-foreman_proxy which was the real offender.